scripts/data_files/driver_templates/psa_crypto_driver_wrappers.c.jinja

```diff
@@ -2422,4 +2422,90 @@ psa_status_t psa_driver_wrapper_asymmetric_encrypt(
     }
 }
 
+psa_status_t psa_driver_wrapper_asymmetric_decrypt(
+    const psa_key_attributes_t *attributes,
+    const uint8_t *key_buffer,
+    size_t key_buffer_size,
+    psa_algorithm_t alg,
+    const uint8_t *input,
+    size_t input_length,
+    const uint8_t *salt,
+    size_t salt_length,
+    uint8_t *output,
+    size_t output_size,
+    size_t *output_length )
+{
+    psa_status_t status = PSA_ERROR_CORRUPTION_DETECTED;
+    psa_key_location_t location =
+        PSA_KEY_LIFETIME_GET_LOCATION( attributes->core.lifetime );
+
+    switch( location )
+    {
+        case PSA_KEY_LOCATION_LOCAL_STORAGE:
+            /* Key is stored in the slot in export representation, so
+             * cycle through all known transparent accelerators */
+#if defined(PSA_CRYPTO_ACCELERATOR_DRIVER_PRESENT)
+#if defined(PSA_CRYPTO_DRIVER_TEST)
+            status = mbedtls_test_transparent_asymmetric_decrypt( attributes,
+                                                                  key_buffer,
+                                                                  key_buffer_size,
+                                                                  alg,
+                                                                  input,
+                                                                  input_length,
+                                                                  salt,
+                                                                  salt_length,
+                                                                  output,
+                                                                  output_size,
+                                                                  output_length );
+            /* Declared with fallback == true */
+            if( status != PSA_ERROR_NOT_SUPPORTED )
+                return( status );
+#endif /* PSA_CRYPTO_DRIVER_TEST */
+#endif /* PSA_CRYPTO_ACCELERATOR_DRIVER_PRESENT */
+            return( mbedtls_psa_asymmetric_decrypt( attributes,
+                                                    key_buffer,
+                                                    key_buffer_size,
+                                                    alg,
+                                                    input,
+                                                    input_length,
+                                                    salt,
+                                                    salt_length,
+                                                    output,
+                                                    output_size,
+                                                    output_length ) );
+        /* Add cases for opaque driver here */
```

Comment:
Can this be removed?

Reply:
I'm not sure it is also present in other test functions that I was following while working on this.